src/Polly.Pcl/project.json

```diff
@@ -1,12 +1,7 @@
 ﻿{
-    "dependencies": {
-        "NuSpec.ReferenceGenerator": {
-            "version": "1.4.0",
-            "type": "build"
-        }
-    },
-    "frameworks": {
-        ".NETPortable,Version=v4.5,Profile=Profile259": { }
-    },
-    "supports": {}
-}
+  "dependencies": {},
+  "frameworks": {
+    ".NETPortable,Version=v4.5,Profile=Profile259": {}
```

Comment:
Sounds good. Let me know if you have any questions about .NET Standard/VS 2017. btw I'm part of a team in the .NET/VS group whose goals include helping community projects adopt (and adapt) to the changes in the .NET ecosystem and help where needed. I recently learned of this project and have started incorporating it into one of our internal projects :)